### DISCUSSION TITLE
Add action for opening universal/deep links

### DISCUSSION THREAD

**Discussion by Andreyco** [fastlane/fastlane#17830] [Ideas]
Universal/deep link support is common task mobile apps handle.

Launching app from such link is repetitive task during development cycle and dev experience could be improved by providing dedicated action, abstracting away underlying platform specific shell commands.

For such use case, I created simple action doing what is described in previous paragraph.
At this point, it's very simple - supporting launching the action against simulator.
I could imagine future expansion by providing interactivity
- providing list of connected devices or launched simulators
- validation of link scheme against manifest files
- more

I'm curious community would accept such action and whether it's worthy to include in base actions.

**Comment by rogerluan** [comment]
Hey @Andreyco 👋 

I'm glad to hear you create an action to help with such time consuming task! 
Although we're unlikely to add new non-essential actions into _fastlane_ main repo, we have a dedicated organization for community-driven plugins and actions: https://github.com/fastlane-community

At this moment I'm not sure how the process works for new action submissions, so I'm gonna ping @joshdholtz to clarify on that 😊 

Do you happen to have your action publicly available in a repo of yours already? 

Looking forward to seeing your new action! 🚀
